queries.sql

```diff
@@ -22,3 +22,65 @@ SELECT (name) FROM animals WHERE name NOT IN ('Gabumon');
 SELECT (name) FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+-- Transaction
+
+SELECT * FROM animals
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon%';
+
+UPDATE animals 
+SET species = 'pokemon'
+WHERE name NOT LIKE '%mon%'; 
+
+COMMIT;
+
+SELECT * FROM animals WHERE species = 'digimon';
+SELECT * FROM animals
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+DELETE FROM animals
+WHERE date_of_birth >= '2022-01-01';
+
+SAVEPOINT SP1;
```

Comment:
- [ ] the `SAVEPOINT SP1;` it should use inside  a transaction because you are using it outside a transaction it gives the following error please make sure to fix it 
`ERROR:  SAVEPOINT can only be used in transaction blocks`